### DISCUSSION TITLE
Weird stutter on SteamVR

### DISCUSSION THREAD

**Discussion by KavExtraspace** [DaXcess/LCVR#172] [Q&A] [answered]
When i first got the mod it was right before it was published with my friend who plays on quest 2 as-well but on link cable and he said there was this sickening stutter that made the experience so bad he just didn't even want to play on screen mode after. Me being on a quest 2 as-well i was quite confused, until i realized, I play on Virtual Desktop (VD) and it turns out i had set my Vr runtime to VDXR and not SteamVr which apparently completely fixes the stutter issue, however my friend can't use VDXR

Please fix this issue. Im not a dev but i do understand this is a passion project and you may not fully know what the problem is or you may not have the time to fix it etc, but it is causing a lot of people to be unable to play (or at-least makes it sickening to) and is very annoying. I have seen work arounds to this problem, however i have not tested them yet since it is only for oculus link users, which i am not one so i cannot. But id like a full fix or patch for this instead of putting it together with some duct tape. Thank you for reading.

I have tested it with no mods
I have tried with the latest version
I have turned off and on settings
The only thing that fixes it is using VDXR or OpenXR, and using that on a link cable, is annoying.

TLDR: apparently VDXR causes no stutter while SteamVR does so i cant play with my friend, please fix.

**Comment by DaXcess** [comment, marked as answer]
> *Forgot that discussions existed*

I'm guessing this was fixed since version v1.1.5 (released Jan 27th, this thread was created three days before, when the issue was still a thing)
